internal/mempool/error_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2014 The btcsuite developers
+// Copyright (c) 2015-2020 The Decred developers
```

Comment:
This is an entirely new file.  The copyright shouldn't be attributed to btcsuite or pre 2020.

```Go
// Copyright (c) 2020 The Decred developers
// Use of ...
```